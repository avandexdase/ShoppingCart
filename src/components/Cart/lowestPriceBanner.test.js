import React from 'react';
import renderer from 'react-test-renderer';
import LowestPriceBanner from './lowestPriceBanner';

it('changes the class when hovered', () => {
  const component = renderer.create(
    <LowestPriceBanner />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});