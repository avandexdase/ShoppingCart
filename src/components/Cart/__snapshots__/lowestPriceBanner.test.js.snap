// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<div
  className="lowestPriceBanner"
>
  <img
    alt="lowest_price"
    src="../../../static/images/lowest-price.png"
  />
  <span>
    You Won't find it cheaper anywhere
  </span>
</div>
`;
